Admin/AbstractAdmin.php

```diff
@@ -422,8 +422,6 @@
      */
     protected $securityInformation = array();
 
-    protected $cacheIsGranted = array();
```

Comment:
> Accessing it could be deprecated mith magic methods if deemed worth it.

Dunno, dou you have some samples?

We can start by adding `@deprecated` annotation for IDE.